lib/twitter-ads/creative/cards_fetch.rb

```diff
@@ -82,26 +81,27 @@ def all(*)
           "'CardsFetch' object has no attribute 'all'")
       end
 
-      def load(account, card_uri = nil, card_id = nil, with_deleted = nil, opts = {})
-        if (card_uri && card_id) || (card_uri.nil? && card_id.nil?)
+      def load(account, card_uris = nil, card_id = nil, with_deleted = nil, opts = {})
+        if (card_uris && card_id) || (card_uris.nil? && card_id.nil?)
           raise ArgumentError.new('card_uri and card_id are exclusive parameters. ' \
```

Comment:
nit: "'card_uris and card_id are exclusive parameters."

Reply:
Good catch! Will update!